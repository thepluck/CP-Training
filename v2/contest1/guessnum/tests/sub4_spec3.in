2
357328
422750
