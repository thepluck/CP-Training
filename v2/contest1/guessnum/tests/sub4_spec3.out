26192773675
36661860081
